firebase.json

```diff
@@ -1171,7 +1171,9 @@
         "source": "/dart-2.0",
         "destination": "/dart-2",
         "type": 301
-      }
+      },
+      { "source": "/install/**", "destination": "/install", "type": 301 },
+      { "source": "/tools/pub/**", "destination": "/toosl/pub", "type": 301 }
```

Comment:
toosl -> tools
(but why add this redirect? it breaks all the URLs under pub, e.g. https://dartlang-org-staging-1.firebaseapp.com/tools/pub/cmd/pub-get)

Reply:
Oops, clearly not the desired effect. Changed to / was meant to be:

```
      { "source": "/tools/pub/installing", "destination": "/tools/pub/environment-variables", "type": 301 }
```